cypress/e2e/form-builder/autocomplete.cy.ts

```diff
@@ -1,69 +0,0 @@
-describe("Test FormBuilder autocomplete props", () => {
```

Comment:
What was the issue here - I know these tests have got some issues in the past.

Maybe move to component test?

Reply:
The test specs that were deleted had their test moved to other similar specs.  For example the tests in this autocomplete spec were moved to the same spec file as 'add an element'.
The reason for the colocating similar tests is that the cypress runner closes and reopens the chrome process between each test spec.  So by colocating similar tests we save a bit of run time.